packages/vuetify/src/components/VDataTable/VDataTable.tsx

```diff
@@ -164,6 +166,9 @@ export const VDataTable = genericComponent<new <T extends readonly any[], V>(
 
     const paginatedItemsWithoutGroups = computed(() => extractRows(paginatedItems.value))
 
+    const { smAndDown } = useDisplay()
+    const mobileView = computed(() => typeof props.mobile !== 'undefined' ? props.mobile : smAndDown.value)
```

Comment:
If you pass props to useDisplay, you can have mobileBreakPoint handled for you.

Reply:
https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/composables/display.ts#L216

Reply:
Pushed a change for this, let me know if I did this correctly and/or what you were looking for.